acceptance-tests/helpers/servicekeys/get.go

```diff
@@ -1,30 +1,36 @@
 package servicekeys
 
 import (
-	"csbbrokerpakgcp/acceptance-tests/helpers/cf"
 	"encoding/json"
+	"fmt"
 	"reflect"
-	"strings"
+	"regexp"
+
+	"csbbrokerpakgcp/acceptance-tests/helpers/cf"
 
 	. "github.com/onsi/gomega"
 )
 
 func (s *ServiceKey) Get(receiver any) {
 	Expect(reflect.ValueOf(receiver).Kind()).To(Equal(reflect.Ptr), "receiver must be a pointer")
-	out, _ := cf.Run("service-key", s.serviceInstanceName, s.name)
-	start := strings.Index(out, "{")
-	Expect(start).To(BeNumerically(">", 0), "could not find start of JSON")
-	data := []byte(out[start:])
-
-	if cf.Version() == cf.VersionV8 {
-		var wrapper struct {
-			Credentials any `json:"credentials"`
-		}
-		err := json.Unmarshal(data, &wrapper)
-		Expect(err).NotTo(HaveOccurred())
-		data, err = json.Marshal(wrapper.Credentials)
+	keyGUID, errStr := cf.Run("service-key", s.serviceInstanceName, s.name, "--guid")
+	Expect(errStr).To(BeEmpty())
+	nonGUID := regexp.MustCompile(`[^a-zA-Z0-9-]`)
+	keyGUID = nonGUID.ReplaceAllString(keyGUID, "")
+	keyURL := fmt.Sprintf("/v3/service_credential_bindings/%s/details", keyGUID)
+	keyJSON, _ := cf.Run("curl", keyURL)
+	if errStr != "" {
```

Comment:
Looks like this errStr should have been reassigned in the previous line, otherwise it will always be empty as per line 17